### DISCUSSION TITLE
How can I move a molecule to another approximate location

### DISCUSSION THREAD

**Discussion by riversdark** [rdkit/rdkit#5566] [Q&A]
I have an anzyme-substrate complex, with the substrate at its active site. I now want to test the catalytic activity of the enzyme with another substrate, so I need to move the new substrate to where the original substrate is. But of course I can't move the new one to the exactly same location because they are different molecules. How can I do this with RDkit?

I tried moving the new substrate with PyMol but somehow the relative position of the atoms seems to be destroyed because when moved to the new location the atoms formed new and different chemical bonds.

**Comment by kienerj** [comment]
Mabye you can try with open3dalign (which is part of rdkit) but that will require at least some similarity between the substrates and I'm not sure how exactly you would do it also in the context of pymol. So consider this a very high level suggestion.
